### PR TITLE
feat: add login

### PR DESCRIPTION
**Pull request by sagargoel1911**
image

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,20 +1,38 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { SafeAreaView, StatusBar, StyleSheet, Text, View } from 'react-native';
```
Comment (atulpatel-03) [inline App.tsx:1]: Library imports on top, a space and then rest imports

File: src/screens/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Text, View } from 'react-native';
```
Comment (atulpatel-03) [inline src/screens/Login/Login.tsx:1]: Library imports on top, a space and then rest imports

File: src/screens/Login/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import theme from '../../utils/theme';
+import Header from './components/Header';
+import Content from './components/Content';
+
+const Login = () => {
+	return (
+		<View style={{ flex: 1, backgroundColor: theme.colors.brand_color_dark }}>
```
Comment (atulpatel-03) [inline src/screens/Login/Login.tsx:8]: remove inline styling.......use stylesheet

File: src/screens/Login/components/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Image, Pressable, StyleSheet, Text, View } from 'react-native';
```
Comment (atulpatel-03) [inline src/screens/Login/components/Header.tsx:2]: Library imports on top, a space and then rest imports

File: App.tsx (hunk 1 above)
Comment (sagargoel1911) [reply App.tsx]: Resolved

File: src/screens/Login/Login.tsx (hunk 2 above)
Comment (sagargoel1911) [reply src/screens/Login/Login.tsx]: Resolved

File: src/screens/Login/Login.tsx (hunk 3 above)
Comment (sagargoel1911) [reply src/screens/Login/Login.tsx]: Resolved

File: src/screens/Login/components/Content.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import { Button, Pressable, StyleSheet, Text, TextInput, View } from 'react-native';
```
[the remark this answers is not in the document]
Comment (sagargoel1911) [reply src/screens/Login/components/Content.tsx]: Resolved

File: src/screens/Login/components/Header.tsx (hunk 4 above)
Comment (sagargoel1911) [reply src/screens/Login/components/Header.tsx:1]: Resolved

**Pull request merged**